pkg/operator/ceph/cluster/nodedaemon/exporter.go

```diff
@@ -119,6 +119,11 @@ func (r *ReconcileNode) createOrUpdateCephExporter(node corev1.Node, tolerations
 		if cephVersion != nil {
 			controller.AddCephVersionLabelToDeployment(*cephVersion, deploy)
 		}
+
+		// wait for previous exporter pod to be deleted, before creating a new one
+		// to avoid fighting for the same socket file
+		deploy.Spec.Strategy.Type = appsv1.RecreateDeploymentStrategyType
```

Comment:
The change looks good. Don't think it should affect the exporter deployment in any way if we change the deployment strategy. But still curious why this only fails in the provider HCI environment and issue is not seen in other environments